src/__test__/components/AddAssetComponent.test.js

```diff
@@ -189,6 +189,32 @@ describe('<AddAssetContainer />', () => {
     expect(wrapper.state().specs.memory).toEqual('16');
   });
 
+  it('should submit the form when form is filled', () => {
+    specsComponentSetup();
+    const returnSpecsState = () => wrapper.state().specs;
+    wrapper.setProps({
+      createAsset: () => returnSpecsState()
+    });
+
+    expect(returnSpecsState()).toEqual(wrapper.state().specs);
+
+    wrapper.find('DropdownComponent .item').at(3).simulate('click');
+    wrapper.find('input[type="radio"][name="processorType"]').at(0).simulate('change');
+    wrapper.find('input[type="radio"][name="processorSpeed"]').at(1).simulate('change');
+    wrapper.find('input[type="radio"][name="screenSize"]').at(1).simulate('change');
+    wrapper.find('input[type="radio"][name="storage"]').at(2).simulate('change');
+    wrapper.find('input[type="radio"][name="memory"]').at(2).simulate('change');
+    wrapper.find('Form').simulate('submit');
+    expect(wrapper.state().specs).toEqual({
```

Comment:
Why not use `returnSpecsState` here?

Reply:
I don't why I did not add this when committing. I will use `returnSpecsState`